README.md

```diff
@@ -44,15 +44,41 @@ To run checks against an assignment, use the `gator grade` command.
 
 #### Interpreting Output
 
-Output team
+A dictionary will come from the input team that mostly will contain list,
+unpack the dictionary, run through gatorgrade, and return as a list of tuples.
```

Comment:
This sentence is a little unclear. Is it saying that the dictionary coming from the input team will do the things listed? If possible, I would recommend rephrasing for clarity.